tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -40,6 +40,42 @@
 #include <regex>
 
 namespace {
+using AttributeKey_t = ROOT::Experimental::Detail::RDaosContainer::AttributeKey_t;
+using DistributionKey_t = ROOT::Experimental::Detail::RDaosContainer::DistributionKey_t;
+
+/// \brief RNTuple page-DAOS mappings
+enum EDaosMapping { kOClusterDColumn, kOUnique };
+
+struct RDaosKey {
+   daos_obj_id_t oid;
```

Comment:
Here we should follow ROOT's coding style, i.e. struct/class members should be named `fXxx`.
```suggestion
   daos_obj_id_t fOid;
```